Lib/test/test_codecs.py

```diff
@@ -3078,9 +3111,21 @@ def test_multibyte_encoding(self):
     def test_code_page_decode_flags(self):
         # Issue #36312: For some code pages (e.g. UTF-7) flags for
         # MultiByteToWideChar() must be set to 0.
+        if support.verbose:
+            sys.stdout.write('\n')
         for cp in (50220, 50221, 50222, 50225, 50227, 50229,
                    *range(57002, 57011+1), 65000):
-            self.assertEqual(codecs.code_page_decode(cp, b'abc'), ('abc', 3))
+            # On small versions of Windows like Windows IoT
+            # not all codepages are present.
+            # A missing codepage causes an OSError exception
+            # so check for the codepage before decoding
+            if is_code_page_present(cp):
+                if support.verbose:
+                    sys.stdout.write("  testing cp={}\n".format(cp))
+                self.assertEqual(codecs.code_page_decode(cp, b'abc'), ('abc', 3))
```

Comment:
No need for the verbose message here, but add the codepage value as the third argument of this assertEqual:

```suggestion
                self.assertEqual(codecs.code_page_decode(cp, b'abc'), ('abc', 3), f'cp{cp}')
```